### PR TITLE
5.6.42-28.30 release notes

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

File: doc/source/release-notes/Percona-XtraDB-Cluster-5.6.42-28.30.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+Binaries are available from the `downloads section
+<http://www.percona.com/downloads/Percona-XtraDB-Cluster-56/>`_
+or from our :ref:`software repositories <installation>`.
+
+Percona XtraDB Cluster 5.6.42-28.30 is now the current release,
+based on the following:
+
+* `Percona Server 5.6.42 <https://www.percona.com/doc/percona-server/5.6/release-notes/Percona-Server-5.6.42-84.2.html>`_
+
+* Codership WSREP API release 5.6.41
```
Comment (kbauskar) [inline doc/source/release-notes/Percona-XtraDB-Cluster-5.6.42-28.30.rst]: would be 5.6.42 now.

File: doc/source/release-notes/Percona-XtraDB-Cluster-5.6.42-28.30.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+* Codership Galera library 3.24
```
Comment (kbauskar) [inline doc/source/release-notes/Percona-XtraDB-Cluster-5.6.42-28.30.rst]: would be 3.25 now.

File: doc/source/release-notes/Percona-XtraDB-Cluster-5.6.42-28.30.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
+.. rn:: 5.6.42-28.30
+
+===================================
+Percona XtraDB Cluster 5.6.42-28.30
+===================================
+
+Percona is glad to announce the release of
+Percona XtraDB Cluster 5.6.42-28.30 on December 20, 2018.
```
Comment (kbauskar) [inline doc/source/release-notes/Percona-XtraDB-Cluster-5.6.42-28.30.rst]: please set date as 4th Jan 2019

**Review by kbauskar — CHANGES_REQUESTED** [review]

**Review by hrvojem — CHANGES_REQUESTED** [review]
bump the version in conf.py

**Review by kbauskar — APPROVED** [review]

**Review by hrvojem — APPROVED** [review]
LGTM

**Pull request merged**